Protocol.md

```diff
@@ -245,22 +239,22 @@ When the max Stream ID has been used:
 
 1) If Stream ID re-use is not employed:
 
-- No new streams can be created, thus a new connection MUST be established to create new streams once the max has been met. 
+    - No new streams can be created, thus a new connection MUST be established to create new streams once the max has been met. 
 
 2) If Stream IDs are re-used:
 
-- The Requester MUST re-use IDs by wrapping and restarting at ID 1 for client, and ID 2 for server, and incrementing sequentially by 2s as stated above.
-- The Requester MUST skip IDs still in use. 
-- The Responder MAY choose to ERROR[REJECT] any Stream ID if it considers the ID still in use. The Requester MAY retry on the next sequential Stream ID considered unused.
-- If all Stream IDs are concurrently in use, no new streams can be created, thus a new connection MUST be established to create new streams.
+    - The Requester MUST re-use IDs by wrapping and restarting at ID 1 for client, and ID 2 for server, and incrementing sequentially by 2s as stated above.
+    - The Requester MUST skip IDs still in use. 
+    - The Responder MAY choose to ERROR[REJECT] any Stream ID if it considers the ID still in use. The Requester MAY retry on the next sequential Stream ID considered unused.
+    - If all Stream IDs are concurrently in use, no new streams can be created, thus a new connection MUST be established to create new streams.
 
 It is RECOMMENDED that Stream ID re-use only be used in combination with resumability. 
 
 ### Frame Types
 
 |  Type                          | Value  | Description |
 |:-------------------------------|:-------|:------------|
-| __RESERVED__                                     | 0x00 | __Reserved__ |
+| __RESERVED__                                     | 0x00 | __Reserved__ for any operation involving the connection. |
```

Comment:
This is not correct.  `Stream ID of value __0__` but this section is discussing the frame types, not stream ids.

Reply:
oh !!!! got it
```suggestion
| __RESERVED__                                     | 0x00 | __Reserved__ |
```

Reply:
oh !!! got it 👍🏽